flexget/plugins/cli/wait_list.py

```diff
@@ -0,0 +1,247 @@
+                return
+        approve_text = 'approved' if approve else 'rejected'
+        if (db_entry.approved is True and approve is True) or (db_entry.approved is False and approve is False):
+            console('entry {} is already {}'.format(db_entry.title, approve_text))
+            return
+        db_entry.approved = approve
+        console('Successfully marked pending entry {} as {}'.format(db_entry.title, approve_text))
+
+
+def wait_list_del(options):
+    with Session() as session:
+        try:
+            entry_list = get_list_by_exact_name(options.list_name)
+        except NoResultFound:
+            console('Could not find wait list with name `{}`'.format(options.list_name))
+            return
+        try:
+            db_entry = get_wait_entry_by_id(entry_list.id, int(options.entry), session=session)
+        except NoResultFound:
+            console(
+                'Could not find matching wait entry with ID {} in list `{}`'.format(int(options.entry),
+                                                                                    options.list_name))
+            return
+        except ValueError:
+            db_entry = get_wait_entry_by_title(entry_list.id, options.entry, session=session)
+            if not db_entry:
+                console(
+                    'Could not find matching wait entry with title `{}` in list `{}`'.format(options.entry,
+                                                                                             options.list_name))
+                return
+        console('Removing wait entry `{}` from list {}'.format(db_entry.title, options.list_name))
+        session.delete(db_entry)
+
+
+def wait_list_purge(options):
+    with Session() as session:
+        try:
+            entry_list = get_list_by_exact_name(options.list_name)
+        except NoResultFound:
+            console('Could not find entry list with name `{}`'.format(options.list_name))
+            return
+        console('Deleting list {}'.format(options.list_name))
+        session.delete(entry_list)
+
+
+@event('options.register')
+def register_parser_arguments():
+    # Common option to be used in multiple subparsers
+    entry_parser = ArgumentParser(add_help=False)
+    entry_parser.add_argument('entry_title', help="Title of the entry")
+    entry_parser.add_argument('url', help="URL of the entry")
+
+    global_entry_parser = ArgumentParser(add_help=False)
+    global_entry_parser.add_argument('entry', help='Can be entry title or ID')
+
+    attributes_parser = ArgumentParser(add_help=False)
+    attributes_parser.add_argument('--attributes', metavar='<attributes>', nargs='+', type=attribute_type,
+                                   help='Can be a string or a list of string with the format imdb_id=XXX,'
+                                        ' tmdb_id=XXX, etc')
+    list_name_parser = ArgumentParser(add_help=False)
+    list_name_parser.add_argument('list_name', nargs='?', default='wait_entries',
```

Comment:
default names for movies_list is "movies" and (hopefully) for series it's "series". This should be "waiting" to be in line?